curator/eslint.config.mjs

```diff
@@ -0,0 +1,23 @@
+import globals from 'globals';
+import pluginJs from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import pluginReact from 'eslint-plugin-react';
+
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  {files: ['**/*.{js,mjs,cjs,ts,jsx,tsx}']},
+  {languageOptions: { globals: {...globals.browser, ...globals.node} }},
+  {rules: {
+    'quotes': ['error', 'single'],
+    // we want to force semicolons
+    'semi': ['error', 'always'],
+    // we use 2 spaces to indent our code
+    'indent': ['error', 2],
+    // we want to avoid extraneous spaces
+    'no-multi-spaces': ['error']
+  }},
```

Comment:
You should prefer [prettier](https://prettier.io/) for formatting. Its more powerful and complete than ts eslint for formatting. However prettier won't catch error in the code, ts-eslint will.

Reply:
Is this something that can be combined with ESLint? If so, what would be the correct way to do it?

Reply:
Have a look in the documentation [here](https://prettier.io/docs/en/integrating-with-linters). 
Small tip, always try to search in the official documentation, there is some documentation that make eslint use prettier directly but it is slow.  Now we disables rules that don't follow prettier rules and let prettier do it's formatting.